content/spin/cache.md

```diff
@@ -36,12 +36,16 @@ These directories are based on the [XDG specification](https://specifications.fr
 
 Installed plugins are stored in `(DATA_DIR)/spin/plugins`.  A snapshot of the plugins registry is also stored under that directory at `(DATA_DIR)/spin/plugins/.spin-plugins`; this is structured as a Git repository.
 
+> Note: If you [install Spin] using Homebrew, the plugins are stored at `$HOMEBREW_PREFIX/fermyon-spin`.
```

Comment:
I think this would be `$HOMEBREW_PREFIX/fermyon-spin/plugins`... @kate-goldenring?

Reply:
I am having a bit of difficulty finding the plugins and templates directory.
I am using the following command to locate where Spin installed:

```bash
$ brew --prefix fermyon/tap/spin
/usr/local/opt/spin
```

But there might be a better way of doing this.